src/main/scala/SBTS3Resolver.scala

```diff
@@ -79,7 +82,22 @@ object SbtS3Resolver extends AutoPlugin {
 
       // convenience method, to use normal bucket addresses with `at`
       // without this resolver: "foo" at s3("maven.bucket.com").toHttps(s3region.value)
-      def toHttps(region: String): String = s"""https://s3-${region}.amazonaws.com/${url.stripPrefix("s3://")}"""
+      def toHttps(region: String): String = {
+        val bucketPath = url.stripPrefix("s3://")
+        val euCentral = Region.EU_Frankfurt.toString
+
+        //Correcting for fact that region enum resolves to null when provided US_STANDARD (us-east-1)
+        val correctedRegion = if(region == null) "us-east-1" else region
```

Comment:
I think this is not a good workaround. A `null` may appear here for another reason and it should be silently replaced with this particular region value. Besides, `US_Standard` is not just a synonym for `US_EAST_1`. From the [AWS Java SDK docs](http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/s3/model/Region.html#US_Standard):

> This is the default Amazon S3 Region. All requests sent to s3.amazonaws.com go to this region unless a location constraint is specified when creating a bucket. The US Standard Region **automatically places data in either Amazon's east or west coast data centers** depending on which one provides the lowest latency.

The `region` argument here is just a string, so you can pass either `"us-east-1"` directly or using the [`Regions`](http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/regions/Regions.html) enum or some other way. My point is that this is not an issue this plugin should solve.